pkg/kudoctl/cmd/install/install.go

```diff
@@ -127,7 +132,11 @@ func getPackageCRDs(name string, options *Options, repository repo.FrameworkRepo
 
 	packageName := bundleVersion.Name + "-" + bundleVersion.Version
```

Comment:
This one here is `Construct(ing) the package name`

Reply:
I'm planning on cleaning this is up